MISRA.md

```diff
@@ -0,0 +1,72 @@
+# MISRA Compliance (for configNUMBER_OF_CORES > 1)
+
+For now, FreeRTOS-Kernel only conforms [MISRA C:2012](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx) guidelines in SMP part (configNUMBER_OF_CORES > 1), 
+with the deviations listed below. Compliance is checked with Coverity static analysis.
+
```

Comment:
Add the MISRA config reference here.
Please reference this [page](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/tree/main/test/Coverity).

Reply:
Because we don't have misra.config in this repo. I put the content of MISRA config instead.
Please review again.